Odata-docs/TOC.yml

```
@@ -57,6 +57,8 @@
         href: /odata/webapi-8/overview
       - name: Getting started
         href: /odata/webapi-8/getting-started
+      - name: Query options
+        href: /odata/webapi-8/fundamentals/query-options
```

```suggestion
      - name: Fundamentals
        items:
          - name: Query options
            href: /odata/webapi-8/fundamentals/query-options
```

I noticed from the doc that @asanyaga did that we should have an item named `Fundamentals` with sub items. **Actions and Functions** should actually have gone under **Fundamentals**. You did it right in terms on the side of folder structure but on the menu you didn't do the same. We can bring **Actions and Functions** under **Fundamentals** in a separate PR. The change wouldn't be a breaking one